run.py

```diff
@@ -793,10 +793,11 @@ def run(self, outdir):
 
             if not check_value["failure"] and not check_value["skipped"]:
                 if not self.quiet:
-                    if os.path.basename(os.path.dirname(self.directory)) != "tests":
-                        path_name = os.path.join(os.path.basename(os.path.dirname(self.directory)), self.name)
+                    test_name_offset = self.directory.find("tests/")
+                    if test_name_offset > -1:
+                        path_name = self.directory[test_name_offset + len("tests/"):]
                     else:
-                        path_name = (os.path.basename(self.directory))
+                        path_name = os.path.basename(self.directory)
```

Comment:
Does not this case need to be updated as well not to use basename ?

Reply:
Perhaps.  But this case is hit when you are pointing Suricata at some directory of private tests.  I didn't want to make any assumptions as to what that directory might look like, and the "top" directory pointed to by the command line is lost by the time we get here.

But perhaps it makes sense to just display the full path then?

```
[jason@desktop …/dev/suricata/master]  master ❯ ~/oisf/dev/suricata-verify/master/run.py --testdir ~/scratch
    Finished release [optimized] target(s) in 0.01s
Number of concurrent jobs: 8
===> /home/jason/scratch/dns-invalid-opcode: OK

PASSED:  1
FAILED:  0
SKIPPED: 0
```

Reply:
full path sounds good to me

Reply:
Will use full-path.